src/views/dashboard/Dashboard.js

```diff
@@ -2,37 +2,37 @@ import React, { useState } from 'react'
 import styled from 'styled-components'
 import { useHistory } from 'react-router-dom'
 import { ViewContainer, ActionsContainer, Fab } from '../../components'
-import { ReactComponent as Calendar } from '../../components/icons/calendar.svg';
-import { ReactComponent as Team } from '../../components/icons/team.svg';
-import { ReactComponent as Services } from '../../components/icons/cardiovascular.svg';
+import { ReactComponent as Calendar } from '../../components/icons/calendar.svg'
+import { ReactComponent as Team } from '../../components/icons/team.svg'
+import { ReactComponent as Services } from '../../components/icons/cardiovascular.svg'
```

Comment:
start looking for a way to handle images outside the repository!

Reply:
TBD

Reply:
A issue was generated an is in the backlog https://github.com/Phlammariont/happiny/issues/19